module/VuFind/src/VuFind/Db/Table/UserListFactory.php

```diff
@@ -67,8 +67,6 @@ public function __invoke(
         if (!empty($options)) {
             throw new \Exception('Unexpected options sent to factory!');
         }
-        $sessionManager = $container->get(\Laminas\Session\SessionManager::class);
-        $session = new \Laminas\Session\Container('List', $sessionManager);
-        return parent::__invoke($container, $requestedName, [$session]);
+        return parent::__invoke($container, $requestedName, [null]);
```

Comment:
The session is no longer used here, so I deleted it to prevent confusion. We could eliminate the whole factory, but that seemed more disruptive (just in case we need to add new dependencies here before the refactoring is done and end up bringing it back).